loqusdb/build_models/variant.py

```diff
@@ -190,7 +196,9 @@ def build_variant(variant, case_obj, case_id=None, gq_threshold=None, gq_qual=Fa
             ind_pos = ind_obj["ind_index"]
 
             if gq_qual:
-                gq = int(variant.QUAL)
+                gq = 0
+                if variant.QUAL:
+                    gq = int(variant.QUAL)
```

Comment:
But this won't solve the problem of providing an invalid literal for int(), right?
```suggestion
                if variant.QUAL:
                    try:
                        gq = int(variant.QUAL)
                    except ValueError:
                        gq = 0
```

Reply:
Ah, no, indeed, thank you kindly for writing it up - but I kind of thought Cyvcf2 would have dealt with the literal `.` already, setting the value to `None` which  conveniently also bools to `False`.

Reply:
I see, neat 👍